homeassistant/components/ntfy/strings.json

```
@@ -250,6 +250,70 @@
     },
     "timeout_error": {
       "message": "Failed to connect to ntfy service due to a connection timeout"
+    },
+    "entity_not_found": {
+      "message": "The selected ntfy entity could not be found."
+    },
+    "entry_not_loaded": {
+      "message": "The selected ntfy service is currently not loaded or disabled in Home Assistant."
+    },
+    "delay_no_email": {
+      "message": "Delayed email notifications are not supported"
+    },
+    "delay_no_call": {
+      "message": "Delayed call notifications are not supported"
+    }
+  },
+  "services": {
+    "publish": {
+      "name": "Publish notification",
+      "description": "Publishes a notification message to a ntfy topic",
+      "fields": {
+        "title": {
+          "name": "[%key:component::notify::services::send_message::fields::title::name%]",
+          "description": "[%key:component::notify::services::send_message::fields::title::description%]"
+        },
+        "message": {
+          "name": "[%key:component::notify::services::send_message::fields::message::name%]",
+          "description": "[%key:component::notify::services::send_message::fields::message::description%]"
+        },
+        "markdown": {
+          "name": "Format as Markdown",
+          "description": "Enable Markdown formatting for the message body (Web app only). See the Markdown guide for syntax details: https://www.markdownguide.org/basic-syntax/."
+        },
+        "tags": {
+          "name": "Tags/Emojis",
+          "description": "Add tags or emojis to the notification. Emojis (using shortcodes like smile) will appear in the notification title or message. Other tags will be displayed below the notification content."
+        },
+        "priority": {
+          "name": "Message priority",
+          "description": "All messages have a priority, which defines how urgently your phone notifies you, depending on the configured vibration patterns, notification sounds, and visibility in notification drawer or pop-over."
```

I'm sorry, I messed this up by editing my first comment.

The first comma needs to go instead (English is just the opposite of German here as well), also "that" is recommended instead of "which" and adding a "the" improves the grammar a bit, too:

```suggestion
          "description": "All messages have a priority that defines how urgently your phone notifies you, depending on the configured vibration patterns, notification sounds, and visibility in the notification drawer or pop-over."
```